app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -191,21 +192,31 @@ class SamplesView extends React.Component {
         ? mapSidebarSelectedSampleIds
         : selectedSampleIds;
 
-    const log = () =>
-      logAnalyticsEvent("SamplesView_heatmap-icon_clicked", {
-        selectedSampleIds: targetSampleIds.length,
+    const heatmapOptions = [
+      { text: "Taxon Heatmap", value: "/visualizations/heatmap" },
+    ];
+    if (this.props.admin) {
+      heatmapOptions.push({
+        text: "AMR Heatmap",
+        value: "/amr_heatmap",
       });
+    }
     return targetSampleIds.size < 2 ? (
       <HeatmapIcon className={cx(cs.icon, cs.disabled, cs.heatmap)} />
     ) : (
-      <a
-        onClick={log}
-        href={`/visualizations/heatmap?sampleIds=${Array.from(
-          targetSampleIds
-        )}`}
-      >
-        <HeatmapIcon className={cx(cs.icon, cs.heatmap)} />
-      </a>
+      <HeatmapIconDropdown
+        className={cx(cs.icon, cs.heatmapDropdown)}
+        iconClassName={cx(cs.icon, cs.heatmap)}
+        options={heatmapOptions}
+        onClick={heatmapOption => {
+          logAnalyticsEvent("SamplesView_heatmap-icon_clicked", {
+            selectedSampleIds: targetSampleIds.length,
+            heatmapOption,
+          });
+          let params = Array.from(targetSampleIds).join("&sampleIds[]=");
+          window.location.assign(`${heatmapOption}?sampleIds[]=${params}`);
```

Comment:
Instead of calling `window.location.assign`, I would use `<a>` tags for each option. See how `UserDropdown` in `Header.jsx` does it. It's better to use `<a>` tags whenever possible, because they have better native browser support. (for example, when the user hovers over the link, the dest address will be displayed at the bottom)